app/models/compute_resources/foreman/model/gce.rb

```diff
@@ -154,7 +154,18 @@ def create_volumes(args)
     end
 
     def available_images
-      client.images
+      images_list = client.images.current
+      if image_families_to_filter.present?
```

Comment:
can this be `nil`? if not:
```suggestion
      if image_families_to_filter.any?
```
sounds bit better